solutions/CF Mixing Water.mdx

```diff
@@ -0,0 +1,132 @@
+---
+id: cf-1359C
+source: CF
+title: Mixing Water
+author: Jesse Choe
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/78116)
+
+**Note: the $\mathcal{O}(1)$ solution for this problem is provided in the
```

Comment:
maybe put this in an info block?

Reply:
How do I do that?

Reply:
https://usaco.guide/general/working-mdx/#info-block